community/doc_guidelines/general.md

```diff
@@ -0,0 +1,83 @@
+# General Guidelines for Splinter Documentation
+
+<!--
+  Copyright 2018-2020 Cargill Incorporated
+  Licensed under Creative Commons Attribution 4.0 International License
+  https://creativecommons.org/licenses/by/4.0/
+-->
+
+* Be consistent.
+
+* Follow the [Splinter capitalization guidelines](capitalization.md).
+
+* For punctuation and font changes, follow [Google's typographic
+  conventions](https://developers.google.com/style/typographical-conventions).
```

Comment:
looks like this link redirects to https://developers.google.com/style/text-formatting any reason not to link directly?

Reply:
Updated this link and others.  

I also made a few other improvements while I was there: added an intro, reorganized for clarity, fixed items that don't follow the doc guidelines, and added an attribution for the Google Developer Documentation Style Guide (per their request).